### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by om64**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any

### CODE REVIEW

File: plugin/blog/Resources/translations/icap_blog.nl.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+icap_blog_infos_form_infos: Informatie
+icap_blog_post_form_publicationDate: Publicatiedatum
+new_post: Nieuw artikel
+no_post: Geen artikel
+add_new_post: Een artikel toevoegen
+edit_post: Het artikel "%postTitle%" bewerken
+back_to_blog: Terug naar de blog
+icap_blog_edit_banner: De banner bewerken
+icap_blog_options_form_authorize_comment: Commentaren over de artikelen toelaten
+icap_blog_options_form_authorize_anonymous_comment: Commentaren door onbekenden toelaten (Geen vereiste authenticatie)'
```
Comment (kitan1982) [inline plugin/blog/Resources/translations/icap_blog.nl.yml]: Trailing quote

**Comment by kitan1982** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM again

**Pull request merged**